mongodbatlas/clusters_test.go

```diff
@@ -28,6 +28,16 @@ func TestClusters_ListClusters(t *testing.T) {
 						"readPreference": "secondary"
 					},
 					"clusterType": "REPLICASET",
+					"connectionStrings": {
+						"standardSrv": "mongodb+srv://cluster0-auylw.mongodb.net",
+						"awsPrivateLink": {
+							"vpce-0d00c26273372c6ef": "mongodb://pl-0-us-east-1-auylw.mongodb.net:1024,pl-0-us-east-1-auylw.mongodb.net:1025,pl-0-us-east-1-auylw.mongodb.net:1026/?ssl=true&authSource=admin&replicaSet=Cluster0-shard-0-shard-0"
+						},
+						"awsPrivateLinkSrv": {
+							"vpce-0d00c26273372c6ef": "mongodb+srv://cluster0-pl-0-auylw.mongodb.net"
+						},
+						"standard": "mongodb://cluster0-shard-00-00-auylw.mongodb.net:27017,cluster0-shard-00-01-auylw.mongodb.net:27017,cluster0-shard-00-02-auylw.mongodb.net:27017/?ssl=true&authSource=admin&replicaSet=Cluster0-shard-0"
```

Comment:
[nit] Any reason not to put this above `standardSrv` to make it more readable?

Reply:
I used copy&paste from the example in docs 😬, but I can change it anyway 👍